keyboards/preonic/keymaps/rs/keymap.c

```diff
@@ -0,0 +1,63 @@
+#include QMK_KEYBOARD_H
```

Comment:
Actually, if you add the community layout support to your board, and move this keymap into `/layouts/community/ortho_5x12/`, it would be used for both boards.